strapi/sdk-plugin
# Add automatic install and build for plugins

### What does it do?

- Made it so we install and build plugins
- we use the preferred package manager or the explicitly chosen package manager

### How to test it?

- if you run the init command in a strapi project:
   - the strapi project package manager will be used

- if you use the `--use-yarn`, `--use-pnpm` or `--use-npm` :
   - the chosen package manager will be used 

- you use `--no-install` to avoid installing (it installs and builds by default)

- your preferred system packageManager will be used automatically

running those commands should be successful

### Related issue(s)/PR(s)

fixes #34
